src/App.js

```diff
@@ -56,8 +72,8 @@ export default function App() {
     </>
   )
 
-  function CreateTrip(newCard) {
-    setCards([newCard, ...cards])
+  function CreateTrip(newTripCard) {
+    setTripCards([newTripCard, ...tripCards])
   }
 
   function saveToLocal(key, data) {
```

Comment:
The `saveToLocal` and `loadFromLocal` functions should be in a `services` folder.

Reply:
Done! 👍🏻